snapcraft/file_utils.py

```diff
@@ -264,6 +264,9 @@ def create_similar_directory(source: str, destination: str) -> None:
     gid = stat.st_gid
     os.makedirs(destination, exist_ok=True)
 
+    if sys.platform == "win32":
```

Comment:
Might be a good idea to add the commit description or similar as a code comment

Reply:
Good call, fixed!